2024-11-25 09:56:02  [ Thread-3:0 ] - [ ERROR ]  Task failed:java.util.concurrent.ExecutionException: java.lang.IllegalArgumentException: url:https://127.0.0.1/mobile/inc/get_contactlist.php?P=1&KWORD=%25&isuser_info=3发生错误:java.net.ConnectException: Connection refused: connect , retrying...1
2024-11-25 09:56:06  [ Thread-3:3242 ] - [ ERROR ]  Task failed:java.util.concurrent.ExecutionException: java.lang.IllegalArgumentException: url:https://127.0.0.1/mobile/inc/get_contactlist.php?P=1&KWORD=%25&isuser_info=3发生错误:java.net.ConnectException: Connection refused: connect , retrying...2
2024-11-25 09:56:09  [ Thread-3:6502 ] - [ ERROR ]  Task failed:java.util.concurrent.ExecutionException: java.lang.IllegalArgumentException: url:https://127.0.0.1/mobile/inc/get_contactlist.php?P=1&KWORD=%25&isuser_info=3发生错误:java.net.ConnectException: Connection refused: connect , retrying...3
2024-11-25 09:56:10  [ Thread-3:7504 ] - [ ERROR ]  Error occurred while executing task: java.lang.RuntimeException: Task failed after 3 retries
2024-11-25 09:56:12  [ Thread-3:9738 ] - [ ERROR ]  Task failed:java.util.concurrent.ExecutionException: java.lang.IllegalArgumentException: url:https://127.0.0.1/general/mytable/intel_view/video_file.php?MEDIA_DIR=../../../inc/&MEDIA_NAME=oa_config.php发生错误:java.net.ConnectException: Connection refused: connect , retrying...1
2024-11-25 09:56:15  [ Thread-3:12997 ] - [ ERROR ]  Task failed:java.util.concurrent.ExecutionException: java.lang.IllegalArgumentException: url:https://127.0.0.1/general/mytable/intel_view/video_file.php?MEDIA_DIR=../../../inc/&MEDIA_NAME=oa_config.php发生错误:java.net.ConnectException: Connection refused: connect , retrying...2
2024-11-25 09:56:19  [ Thread-3:16239 ] - [ ERROR ]  Task failed:java.util.concurrent.ExecutionException: java.lang.IllegalArgumentException: url:https://127.0.0.1/general/mytable/intel_view/video_file.php?MEDIA_DIR=../../../inc/&MEDIA_NAME=oa_config.php发生错误:java.net.ConnectException: Connection refused: connect , retrying...3
2024-11-25 09:56:20  [ Thread-3:17241 ] - [ ERROR ]  Error occurred while executing task: java.lang.RuntimeException: Task failed after 3 retries
2024-11-25 09:56:22  [ Thread-3:19475 ] - [ ERROR ]  Task failed:java.util.concurrent.ExecutionException: java.lang.IllegalArgumentException: url:https://127.0.0.1/general/login_code.php发生错误:java.net.ConnectException: Connection refused: connect , retrying...1
2024-11-25 09:56:25  [ Thread-3:22209 ] - [ ERROR ]  Interrupted exception: Task was interrupted
